### PR TITLE
Erp fed integration

### PR DESCRIPTION
**Pull request by josedahlquist**
*rskj:erp-fed-integration*
*fit:adapt-test-for-erp*

### CODE REVIEW

**Comment by donequis** [comment]
Same issue on `gradle/verification-metadata.xml`

File: gradle/verification-metadata.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <verification-metadata xmlns="https://schema.gradle.org/dependency-verification" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="https://schema.gradle.org/dependency-verification https://schema.gradle.org/dependency-verification/dependency-verification-1.0.xsd">
    <configuration>
-      <verify-metadata>true</verify-metadata>
+      <verify-metadata>false</verify-metadata>
```
Comment (donequis) [inline gradle/verification-metadata.xml:4]: Do we want this?

**Comment by marcos-iov** [comment]
pipeline:run

**Comment by marcos-iov** [comment]
pipeline:run

**Comment by marcos-iov** [comment]
pipeline:run

**Comment by marcos-iov** [comment]
pipeline:run

**Comment by marcos-iov** [comment]
pipeline:run

**Comment by marcos-iov** [comment]
pipeline:run

**Comment by marcos-iov** [comment]
pipeline:run

**Review by marcos-iov — APPROVED** [review]

**Pull request merged**